ecosystem/sep-0010.md

```diff
@@ -113,7 +113,7 @@ On success the endpoint must return `200 OK` HTTP status code and a JSON object
   * time bounds: `{min: now(), max: now() + 900 }` (we recommend expiration of 15 minutes to give user time to sign transaction)
   * operations:
     * `manage_data(source: client_account, key: '<home domain> auth', value: random_nonce())`
-      * The value of key is the home domain the challenge is authenticating, followed by `auth`. It can be at most 64 characters.
+      * The value of key is the home domain the client would like to authenticating with, followed by `auth`. It can be at most 64 characters.
```

Comment:
Can we keep this sentence present tense? I don't think that needs to change to address the issue.
```suggestion
      * The value of key is the home domain the client is authenticating with, followed by `auth`. It can be at most 64 characters.
```